src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/Common/TensorPrimitives.ITernaryOperator.cs

```diff
@@ -19,6 +19,15 @@ private interface ITernaryOperator<T>
             static abstract Vector512<T> Invoke(Vector512<T> x, Vector512<T> y, Vector512<T> z);
         }
 
+        private readonly struct InvertYZTernaryOperator<TOperator, T> : ITernaryOperator<T>
```

Comment:
Should we call this something like `swap` instead of `invert`?

Invert makes me think `-x` or ~x`, as those are the typically "inverses" of the values

Reply:
Ok, I'll rename